client/src/components/questions/AnswerModal.jsx

```diff
@@ -79,13 +82,17 @@ const AnswerModal = ({ productName, questionBody, submit, setAnswerModal }) => {
       return;
     }
 
-    var reader = new FileReader();
+    setPhotoArray(photoArray => [...photoArray, URL.createObjectURL(input.target.files[0])]);
+
```

Comment:
Is this commenting a possible solution using what Karissa suggested?

Reply:
It's actually commenting out my initial solution to creating the uploaded image URL, I am currently using Karissa's solution. Depending on what I hear next week about what the API should be expecting, I can change which version I need to use.